compiler/test/dotc/pos-test-pickling.blacklist

```diff
@@ -43,3 +43,6 @@ i7757.scala
 
 # splice type tag dealiased in this reference
 i8651b.scala
+
+# suspended macros need multiple runs
+i9252
```

Comment:
This macro tests should be under `tests/pos-macros` and this should not be needed in the blacklist. Otherwise LGTM.